tests/beaker_tests/cisco_vpc_domain/test_vpc_domain.rb

```diff
@@ -73,8 +73,10 @@
 # tests[:show_cmd] - the common show command to use for test_show_run
 #
 tests = {
```

Comment:
Please add any platform checks needed to skip properties that are not suported on `n8k`